app/controllers/api/v1/users_controller.rb

```diff
@@ -120,4 +120,24 @@ def update
     standard_update(User, current_human_user.id)
   end
 
-end
\ No newline at end of file
+  ###############################################################
+  # find_unclaimed
+  ###############################################################
+
+  api :POST, '/unclaimed/:email', 'Finds or creates a user by email address.'
```

Comment:
The name of this endpoint is a little funky to me, in that it doesn't necessarily return an unclaimed user.

Do we instead want `application_users/find/email/:email?force=true` or something to that effect? (`force=true` is maybe not the best)

cc @Dantemss